rename_model.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [[ $# -lt 2 ]] || [[ "$@" == "--help" ]] || [[ "$@" == "-h" ]]; then
+    echo "Usage: $0 s_suffix t_suffix"
+    echo " Example: $0 rrfs aqm"
+    exit 0
+fi
+
+# directories to look for model variables to rename
+mdirs=( "docs" "jobs" "parm" "scripts" "test" "tests" "ush" )
+
+# model variables to rename
+mvars=( "home" "parm" "ush" "sorc" "modules" "exec" "scripts" "jobs" "fix" "tests" "we2e" )
```

Comment:
Most of those were in the format `EXECDIR` etc, and I thought that every subdirectory that is under the root directory of UFS SW app need to be named consistently so that renaming the app would be easier. From todays meeting, i understood that it is only `HOMErrfs` that NCO requires the rest can be `PARM`, `USH` etc